Adjust the generic condor_config to Gentoo's needs.
Ported patch from htcondor-8.0.0 to 8.3.2

Patch by Marius Brehler.
--- b/src/condor_examples/condor_config.generic
+++ a/src/condor_examples/condor_config.generic
@@ -19,7 +19,7 @@
 ######################################################################
 
 ##  Where have you installed the bin, sbin and lib condor directories?   
-RELEASE_DIR = /usr/local/condor
+RELEASE_DIR = /usr
 
 ##  Where is the local condor directory for each host?  This is where the local config file(s), logs and
 ##  spool/execute directories are located. this is the default for Linux and Unix systems.
@@ -27,18 +27,22 @@ RELEASE_DIR = /usr/local/condor
 ##  this is the default on Windows sytems
 #LOCAL_DIR = $(RELEASE_DIR)
 
-##  Where is the machine-specific local config file for each host?
-LOCAL_CONFIG_FILE = $(LOCAL_DIR)/condor_config.local
-##  If your configuration is on a shared file system, then this might be a better default
-#LOCAL_CONFIG_FILE = $(RELEASE_DIR)/etc/$(HOSTNAME).local
+## Looking for LOCAL_CONFIG_FILE? You will not find it here. Instead
+## put a file in the LOCAL_CONFIG_DIR below. It is a more extensible
+## means to manage configuration. The order in which configuration
+## files are read from the LOCAL_CONFIG_DIR is lexicographic. For
+## instance, config in 00MyConfig will be overridden by config in
+## 97MyConfig.
 ##  If the local config file is not present, is it an error? (WARNING: This is a potential security issue.)
-#REQUIRE_LOCAL_CONFIG_FILE = true
+REQUIRE_LOCAL_CONFIG_FILE = false
 
-##  The normal way to do configuration with RPMs is to read all of the
-##  files in a given directory that don't match a regex as configuration files.
+##  Where are optional machine-specific local config files located?
 ##  Config files are read in lexicographic order.
-LOCAL_CONFIG_DIR = $(LOCAL_DIR)/config
-#LOCAL_CONFIG_DIR_EXCLUDE_REGEXP = ^((\..*)|(.*~)|(#.*)|(.*\.rpmsave)|(.*\.rpmnew))$
+##  No default.
+LOCAL_CONFIG_DIR        = $(ETC)/config.d
+
+## Blacklist for file processing in the LOCAL_CONFIG_DIR
+LOCAL_CONFIG_DIR_EXCLUDE_REGEXP = ^((\..*)|(.*~)|(#.*)|(.*\.rpmsave)|(.*\.rpmnew))$
 
 ##  Use a host-based security policy. By default CONDOR_HOST and the local machine will be allowed
 use SECURITY : HOST_BASED
